### DISCUSSION TITLE
Response headers not showing as bold

### DISCUSSION THREAD

**Discussion by toxcct** [curl/curl#12103] [Q&A] [answered]
Hello,

My question may sound dumb as this is a pretty old topic now, and many have already covered it, but yet I cannot understand what is wrong in my setup, so please be kind with me 😉

I'm currently curl on Windows, but within the "git bash" terminal provided with the "Git for Windows" package.
My version of curl is:
``` bash
$ curl -V
curl 8.2.1 (x86_64-w64-mingw32) libcurl/8.2.1 Schannel zlib/1.3 brotli/1.0.9 zstd/1.5.5 libidn2/2.3.4 libpsl/0.21.2 (+libidn2/2.3.3) libssh2/1.11.0
Release-Date: 2023-07-26
Protocols: dict file ftp ftps gopher gophers http https imap imaps ldap ldaps mqtt pop3 pop3s rtsp scp sftp smb smbs smtp smtps telnet tftp
Features: alt-svc AsynchDNS brotli HSTS HTTPS-proxy IDN IPv6 Kerberos Largefile libz NTLM PSL SPNEGO SSL SSPI threadsafe UnixSockets zstd
```
And my terminal supports bold obviously, as I can see with a simple echo command:
![image](https://github.com/curl/curl/assets/2319209/5558e314-2d8f-4822-ab31-aa6280eee7b7)

However, even with the switch `--styled-output`, nothing seems to be bolder in curl responses:
![image](https://github.com/curl/curl/assets/2319209/ef2dc8c3-3329-4c9f-bc3a-a7595e11d1d6)

Am I missing something, or is it just that build version of mine that does not output the relevant escape sequences ?

Thank you very much for considering my humble question.

Kind regards,
toxcct

**Comment by dfandrich** [comment, marked as answer]
The man page provides the answer:

> Styled output requires a terminal that supports bold fonts. This feature is
> not present on curl for Windows due to lack of this capability.

**Reply by toxcct** [reply]
First of all, thanks for pointing this out. I didn't see that sentence in the doc.

Well, too bad, because obsiously, the terminal I use *can* display bold chars...
Is it something that was true at the time when this feature has been implemented and may have evolved now to the better, or are there other technical issues behind this ?

Thanks.

**Comment by dfandrich** [comment]
I'm not a Windows guy and I don't know all the history, but gazing at all the
bugs and issues going around regarding problems with simply getting
internationalized domain names working proper when entered from a Windows
terminal, it sounds like its terminal modes can be pretty inconsistent. If you
can come up with a reliable way for this to work on Windows, regardless of the
terminal operating mode or code page, and whether or not it's a ENABLE_UNICODE
build or not, I'm sure a PR would be welcome.
